doc/manual.rst

```diff
@@ -5453,6 +5468,145 @@ powerful programming construct that still suffices. So the "check list" is:
 (4) Else: Use a macro.
 
 
+Alias Declaration
+=================
+
+An alias declaration creates a symbol that is an alias (alternative name)
+for another symbol and can be used transparently in its place. Any symbol can
+be aliased: templates, modules, types, etc; even overloaded symbols can be
+aliased (see `ast.nkClosedSymChoice`).
+The closest equivalent would be D's [alias](https://dlang.org/spec/declaration.html#alias).
+Examples:
+
+.. code-block:: nim
+  
+  when defined(customEcho): alias myecho = myechoImpl
+  else: alias myecho = echo
+  myecho "foo: ", 12
+  doAssert myecho is proc
+
+  alias
```

Comment:
Is `alias` a keyword now?

Reply:
yes but it's not reflected in https://github.com/nim-lang/Nim/pull/11992 yet; until then I'm resorting to the syntax `const myecho = alias2 echo` instead of `alias myecho = echo`